roles/docker/handlers/main.yml

```diff
@@ -4,10 +4,17 @@
   notify:
     - Docker | reload systemd
     - Docker | reload docker
-    - Docker | pause while Docker restarts
     - Docker | wait for docker
 
+# NOTE(bogdando) reloading systemd daemon disturbs docker, we shall wait for it
 - name : Docker | reload systemd
+  command: /bin/true
+  notify:
+    - Docker | systemd-daemon-reload
+    - Docker | wait for docker
```

Comment:
You'll break initial deployment with this. We haven't even started it for the first time when we run systemd-reload.